ansible/roles/installation/docker/tasks/install.yml

```diff
@@ -74,6 +70,7 @@
     delay: '10'
     timeout: '300'
 
-- name: Finishing Docker installation
+- name: end playbook so user can reboot
+  fail:
+    msg: "Installation of Docker is finished. Please restart you computer and run the same oneliner again."
```

Comment:
I don't find very clear if this is an error or not. What's the particular reason of this fail?

Reply:
What i am not sure about is if we can simply restart docker service afterwards  now so we wont require rebooting anymore?

Reply:
So we originally needed a reboot so that docker would work. I added this and commented out the reboot comment and tried on a fresh VM and it went through, pulled the image, created the container then the icons, I could then run the container via the icons with no reboot [commit](https://github.com/shadow-robot/aurora/pull/546/commits/9714ed00c496cc299c0b24fabc2654d7be5f04d7)